zgw_consumers/admin.py

```diff
@@ -13,9 +13,22 @@
 
 @admin.register(Service)
 class ServiceAdmin(admin.ModelAdmin):
-    list_display = ("label", "api_type", "api_root", "nlx", "auth_type")
+    list_display = (
+        "label",
+        "api_type",
+        "api_root",
+        "nlx",
+        "auth_type",
+    )
```

Comment:
I don't see why this needs to be changed and would also not expect black to touch it